packages/fields/src/components/create-template-part-modal/index.tsx

```diff
@@ -135,9 +128,8 @@ export function CreateTemplatePartModalContents( {
 
 	const defaultTemplatePartAreas = useSelect(
 		( select ) =>
-			// @ts-expect-error getEntityRecord is not typed with unstableBase as argument.
 			select( coreStore ).getEntityRecord< {
-				default_template_part_areas: Array< TemplatePartArea >;
+				default_template_part_areas: UnstableBase[ 'default_template_part_areas' ];
```

Comment:
Does it make sense to export `TemplatePartArea` and use it directly?

Reply:
Done. Thanks